src/helpers/globalCss.ts

```diff
@@ -53,496 +53,261 @@ export const core = css`
 }
     
 :root {
-  /* BASE-COLORS */
```

Comment:
Should we leave these existing colors in for backwards compatibility? I know in my testing views leveraging MDHUI I've referenced them, but I don't know if anybody would be relying on them in real stuff.

Reply:
DIsregard - I was looking at an outdated diff.